src/install_pypi.rs

```diff
@@ -286,9 +286,19 @@ fn stream_python_artifacts(
                         format!("'{}' is not a valid python package name", &pkg_data.name)
                     })?;
 
-                let artifact_name = ArtifactName::from_filename(filename, &name)
+                tracing::warn!("version of is  {filename}, {}", pkg_data.version);
```

Comment:
![image](https://github.com/prefix-dev/pixi/assets/12893423/70b31b23-055b-4393-a225-48c89aac2c9b)
I get this now

Reply:
removed it - thank you